recipes/numba-mpi/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "numba-mpi" %}
+{% set version = "0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/numba-mpi-{{ version }}.tar.gz
+  sha256: a1bc879d5b2e4c64d602bef3fd93b5aedc1db40545f062c35447400d9e26e307
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-scm
+  run:
+    - numpy
+    - numba
+    - mpi4py
+    - python
+
+test:
+  commands:
+    - pip check
```

```suggestion
test:
  imports:
    - numba_mpi
  commands:
    - pip check
```

---

import will not work throwing an error because importing `numba_mpi` requires `python` to be executed with `mpiexec python`. We've tried it here: https://github.com/conda-forge/staged-recipes/runs/8033188887

---

CC @david-zwicker

---

```suggestion
test:
  commands:
    - pip check
    - mpiexec python -c "import numba_mpi"
```
Just do something like this.

---

On Linux, it fails with an error message from `mpiexec` indicating missing ssh client. Google (https://www.cfd-online.com/Forums/openfoam-solving/214644-parallel.html) suggests doing something like `apt install openssh_client` - any hints how to best put it into the yaml file? Thanks!

---

Is that package available on conda? Just add it to the test/requires section. Otherwise: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=yum#yum-requirements-txt

---

Thank you, @carterbox!